spec/prog/kubernetes/kubernetes_cluster_nexus_spec.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require_relative "../../model/spec_helper"
+
+RSpec.describe Prog::Kubernetes::KubernetesClusterNexus do
+  subject(:nx) { described_class.new(Strand.new(id: "8148ebdf-66b8-8ed0-9c2f-8cfe93f5aa77")) }
+
+  let(:project) { Project.create_with_id(name: "default") }
```

Comment:
You can replace `create_with_id` with `create` in all cases.  `create_with_id` is only for backwards compatibility.